### PR TITLE
feat: [#185676804] left-aligned side nav items in profile on small sc…

### PR DESCRIPTION
**Pull request by adidner**
…reens

basically there was a weird edge case where the text for the profile side nav, wrapped and centered itself on very small but not quite tablet sized screens.

Before:
image

After:
image

Unchanged wide screen: 
image

Unchanged Mobile:
image

### Ticket

[Replace with ticket_id](https://www.pivotaltracker.com/story/show/)

### Notes

USWDS doesn't have a field for text alignment unfortunately. 

## Code author checklist

- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation, if necessary
- [X] I have not used any relative imports
- [X] I have checked for and removed instances of unused config from CMS
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config
- [X] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [X] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

File: web/src/styles/components/profile-tab.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.align-text-left {
```
Comment (aloverso) [inline web/src/styles/components/profile-tab.scss:1]: This class already exists, do a global search for `text-align: left` in the project

Comment (somabadri) [reply web/src/styles/components/profile-tab.scss:1]: it's `.tal` in `flex.scss`

**Review by somabadri — APPROVED** [review]
good to merge after addressing Anne's comment.

**Review by cbrunefearless — COMMENTED** [review]
lgtm

**Pull request merged**